CHANGELOG.md

```diff
@@ -4,7 +4,9 @@
 
 ### Enhancements
 
+- Add miden-proving-service and miden-proving-service-client crates (#926).
```

Comment:
nit: I know it wasn't you (was me?) but could you update "Add" to "Added" please

Reply:
Yeah, sure 👍🏼